casper/src/main/resources/MultiSigRevVault.rho

```diff
@@ -56,21 +55,11 @@ in {
     } |
 
     contract MultiSigRevVault(@"deployerAuthKey", deployerId, ret) = {
-      new DeployerIdOps(`rho:rchain:deployerId:ops`),
-          revAddrCh,
-          deployerPubKeyBytesCh in {
+      new DeployerIdOps(`rho:rchain:deployerId:ops`), deployerPubKeyBytesCh in {
         DeployerIdOps!("pubKeyBytes", *deployerId, *deployerPubKeyBytesCh) |
-        for (@deployerPubKeyBytes <- deployerPubKeyBytesCh) {
-          _fromPublicKey!(deployerPubKeyBytes, *ret)
-        }
-      }
-    } |
-
-    contract _fromPublicKey(@publicKey, ret) = {
-      new RevAddress(`rho:rev:address`), revAddrCh in {
-        RevAddress!("fromPublicKey", publicKey, *revAddrCh)|
-        for (@revAddress <- revAddrCh) {
-          @AuthKey!("make", (*_multiSigRevVault, revAddress), *ret)
```

Comment:
This was a bug as well, right? revAddress were used instead of pubkey

Reply:
No, this was not part of the bug. Here we need to create secret for auth key that is checked to execute transaction. It can be REV address but it's not necessary so using public key is also valid.